packages/design-system/src/components/N8nDatatable/Datatable.vue

```diff
@@ -0,0 +1,200 @@
+<script lang="ts">
+import { computed, defineComponent, PropType, ref, useCssModule } from 'vue';
+import {
+	DatatableColumn,
+	DatatableRow,
+	DatatableRowDataType,
+} from '@/components/N8nDatatable/mixins';
+import { getValueByPath } from '../../utils';
+import { useI18n } from '../../composables';
+import N8nSelect from '../N8nSelect';
+import N8nOption from '../N8nOption';
+import N8nPagination from '../N8nPagination';
+
+export default defineComponent({
+	name: 'n8n-datatable',
```

Comment:
are you planning to use this somewhere? executions list? would prefer to use this directly to avoid dead code and that the prop interface works here

Reply:
It'll be used on the Resources List component for variables. I'd rather do it in a separate task so PRs don't get too big.